src/modules/common/models/SelectorItemModel.js

```diff
@@ -2,11 +2,13 @@
 export default class SelectorItemModel {
   _code: string
   _name: string
+  _active: boolean
```

Comment:
I'd rather go for the term `enabled` here which is much more common (although I think we had this discussion in the webapp already^^)
https://developer.apple.com/documentation/uikit/uicontrol/1618217-enabled?language=objc
https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.webcontrols.webcontrol.enabled?view=netframework-4.8#remarks

Reply:
Okay for me.